Dockerfile.cpu

```diff
@@ -18,7 +18,7 @@ RUN echo 'ulimit -c 0' >> ~/.bashrc
 RUN pip install https://intel-extension-for-pytorch.s3.amazonaws.com/ipex_dev/cpu/intel_extension_for_pytorch-2.4.0%2Bgitfbaa4bc-cp310-cp310-linux_x86_64.whl
 
 RUN pip install --upgrade pip \
-    && pip install wheel packaging ninja "setuptools>=49.4.0" numpy
+    && pip install wheel packaging ninja "setuptools>=49.4.0" numpy protobuf
```

Comment:
Could you explain why this is needed?

Reply:
@WoosukKwon, this is just for fixing an error when I tried to do some basic test.

```
  File "/usr/local/lib/python3.10/dist-packages/transformers/models/llama/tokenization_llama.py", line 171, in __init__
    self.sp_model = self.get_spm_processor(kwargs.pop("from_slow", False))
  File "/usr/local/lib/python3.10/dist-packages/transformers/models/llama/tokenization_llama.py", line 203, in get_spm_processor
    model_pb2 = import_protobuf(f"The new behaviour of {self.__class__.__name__} (with `self.legacy = False`)")
  File "/usr/local/lib/python3.10/dist-packages/transformers/convert_slow_tokenizer.py", line 43, in import_protobuf
    raise ImportError(PROTOBUF_IMPORT_ERROR.format(error_message))
ImportError: 
The new behaviour of LlamaTokenizer (with `self.legacy = False`) requires the protobuf library but it was not found in your environment. Checkout the instructions on the
installation page of its repo: https://github.com/protocolbuffers/protobuf/tree/master/python#installation and follow the ones
that match your environment. Please note that you may need to restart your runtime after installation.
```

Reply:
@PHILO-HE I see. Can you check the merge conflict?

Reply:
@WoosukKwon, just rebased.
I just move protobuf installation into requirements-common.txt, assuming this issue is not limited to CPU backend. Please correct me if I thought wrongly.

Reply:
I see. It seems `protobuf` is required by Ray and Tensorizer which are dependencies for the NVIDIA GPU backend, so there's no such a problem. I'm ok with adding it to `requirements-common.txt`.